test/GSN/Context.test.js

```diff
@@ -1,11 +1,14 @@
+const { accounts, contract } = require('@openzeppelin/test-environment');
+const [ sender ] = accounts;
```

Comment:
I find it interesting that the definition of the accounts was moved to the top of the file. Are we sure that we don't want to keep it below, just above the describe?

Reply:
I don't feel strongly either way, but it is true that it will make the migration process more obvious to readers. I think my intent was to keep the destructuring close to the declaration of the variable.